django-stubs/contrib/auth/models.pyi

```diff
@@ -23,9 +24,9 @@ class Permission(models.Model):
     content_type_id: int
     objects: ClassVar[PermissionManager]
 
-    name = models.CharField(max_length=255)
-    content_type = models.ForeignKey(ContentType, on_delete=models.CASCADE)
-    codename = models.CharField(max_length=100)
```

Comment:
Having explicit assignments feels a bit weird in a stub file and was causing issues when testing without the plugin (had error code `var-annotated`)